src/swtpm/swtpm_nvstore.c

```diff
@@ -199,6 +203,7 @@ void SWTPM_NVRAM_Shutdown(void)
         g_nvram_backend_ops->cleanup();
     memset(&filekey, 0, sizeof(filekey));
     memset(&migrationkey, 0, sizeof(migrationkey));
+    SWTPM_NVRAM_LinearFile_Cleanup();
```

Comment:
This should be called via the .cleanup interface function rather than a direct call.

Reply:
makes sense, missed that interface